.github/scripts/generate-helm-docs.sh

```diff
@@ -8,11 +8,15 @@
 # Dependencies:
 # Node >=16
 
+# renovate: datasource=github-tags depName=bitnami-labs/readme-generator-for-helm
```

Comment:
please use github release directly wherever possible. This should enhance the metadata we get in renovate update PRs in the future. It should be possible here, since the releases are named the same as the tags. [Link](https://github.com/bitnami-labs/readme-generator-for-helm/releases)
```suggestion
# renovate: datasource=github-releases depName=bitnami-labs/readme-generator-for-helm
```